<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MicrowaveOven</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="DigitalClock" name="lcdClock">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>450</width>
      <height>100</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="bthClock">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Clock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPower">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMode">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDefrost">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Defrost</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>250</width>
      <height>140</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>140</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="notchesVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DigitalClock</class>
   <extends>QLCDNumber</extends>
   <header location="global">digitalclock.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
